config/packages/doctrine.yaml

```diff
@@ -16,3 +16,30 @@ doctrine:
                 dir: '%kernel.project_dir%/src/Entity'
                 prefix: 'App\Entity'
                 alias: App
+
+when@test:
+    doctrine:
+        dbal:
+            # "TEST_TOKEN" is typically set by ParaTest
+            dbname_suffix: '_test%env(default::TEST_TOKEN)%'
+
+when@prod: &prod
+    doctrine:
+        orm:
+            auto_generate_proxy_classes: false
+            query_cache_driver:
+                type: pool
+                pool: doctrine.system_cache_pool
+            result_cache_driver:
+                type: pool
+                pool: doctrine.result_cache_pool
+
+    framework:
+        cache:
+            pools:
+                doctrine.result_cache_pool:
+                    adapter: cache.app
+                doctrine.system_cache_pool:
+                    adapter: cache.system
+
+when@stage: &prod
```

Comment:
Not a fan of this but symfony new recipes forces us to use this now. This way was suggersted by [nicolas](https://twitter.com/nicolasgrekas/status/1496031524705218565)

Reply:
@alexander-schranz have no opinion about that. but as it forces us todo it 🙆‍♂️